Web/src/components/games/BalloonShake/GamePrep.tsx

```diff
@@ -45,6 +46,19 @@ const GamePrep: React.FC<IProps> = ({
             Start Game!
           </BigButton>
         )}
+    {permission === MotionPermission.DENIED
+        && (
+          <>
+            <ShockedPindaSVG />
+            <h1>
+              Uh Oh!
+            </h1>
+            <p>
+              Permissions to your device&apos;s motion sensor was not granted, so
+              you cannot play this game.
```

Comment:
Are we gonna give them the instructions on how to reset such that they get the prompt again?

Reply:
From @juxd : It's unlikely that user will even enter this state